### DISCUSSION TITLE
Symbols in Text?

### DISCUSSION THREAD

**Discussion by zychonatic** [dialogic-godot/dialogic#2164] [Q&A]
hi,
i have been using godot for 2 months now and started using dialogic 2 recently for my dialogues in my game. i'm now wondering if you can add "symbols" in text like "arrow up" or so to display controls and so on?.

is there a notation for that or any other chance to add symbols like that in the textbox?

thx

**Comment by CakeVR** [comment]
Hello, images are not supported in Godot as of 4.2 if the code is using visibility: https://github.com/godotengine/godot/issues/86697
I came up with an idea and it seems to work, it will be a workaround. I will let you know when the code is done.

**Comment by CakeVR** [comment]
Hello once again, this bug has been fixed. See the same issue again: https://github.com/godotengine/godot/issues/86697

Therefore, I have decided not adding a bypassing fix, but I suggest using 4.3 of Godot instead.
